### ISSUE TITLE
Private browsing by default

### ISSUE THREAD

**Issue by bsclifton** [brave/browser-laptop#5132]
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
From support:
> I would like the ability to make private browsing my default and setting upon opening, etc.

It would be great to allow for an "always private" or "private by default" mode

**Expected behavior:**
There should be a preference to enable this mode which prevents non-private tabs from being created.

- Platform (Win7, 8, 10? macOS? Linux distro?):
all

- Brave Version:
all

**Comment by luixxiul** [comment]
I closed the similar ticket #434 based on the Private Data options, but maybe it should have been not.

**Comment by bsclifton** [comment]
@luixxiul yes- it looks like #434 is requesting the same functionality.  The fix that was mentioned with #2472 does help, but isn't exactly what is requested

**Comment by BrendanEich** [comment]
Would this allow a toggle to disjoint non-private tab-set, as iOS Safari does?

**Comment by flamsmark** [comment]
In general, how is this different from dropping all local state when the browser closes?
